src/main/scala/io/picnicml/doddlemodel/preprocessing/QuantileDiscretizer.scala

```diff
@@ -0,0 +1,106 @@
+package io.picnicml.doddlemodel.preprocessing
+
+import breeze.linalg.DenseVector
+import breeze.stats.DescriptiveStats
+import cats.syntax.option._
+import io.picnicml.doddlemodel.CrossScalaCompat._
+import io.picnicml.doddlemodel.data.Feature.FeatureIndex
+import io.picnicml.doddlemodel.data.Features
+import io.picnicml.doddlemodel.syntax.OptionSyntax._
+import io.picnicml.doddlemodel.typeclasses.Transformer
```

Comment:
Another line between the penultimate import and `import scala.Double.{MaxValue, MinValue}` (I used `Optimize imports` in IntelliJ which also reordered some of the imports).

Reply:
I use Emacs sadly, but I'll open IntelliJ for the import optimization (I have used Scalafix for similar things, but it's a big dependency for something IntelliJ does for free for most folks)